### ISSUE TITLE
Question on the SDK

### ISSUE THREAD

**Issue by lannguyen0910** [visualcamp/seeso-sample-android#19]
Greetings the authors!

This issue is much more a question rather than an issue.
- I would like to know about the SDK lib. Did you use a deep learning model to train for the lib? If yes, may i ask you to give the explanation on it, or the method that you use to build.
- An additional question is that how accurate is the lib? Is there any benchmark to compare the accuracy?

Thank you so much in advance!

Hoang-Lan Nguyen

**Comment by GabeShin** [comment]
Hello @lannguyen0910,

Sorry for the late reply. 
We did use the deep learning model for the SDK. And unfortunately, we can't disclose the details of the model or the algorithm at this point.

We have metrics to compare the accuracy of the models for internal use. We have tested on common public gaze tracking datasets. We are planning to release the research paper comparing our models to other models sometime this year. 

I'll share the paper once it's published. 

Best, 
Gabe
